tests/app/services/test_process_request_json.py

```diff
@@ -162,21 +168,22 @@ def test_missing_field_in_transformation():
                 ]
             }
         },
-    ], TEXT_FIELDS_SET=set(['someField', 'otherField'])):
-        request = {
-            "otherField": ["wibble"],
-        }
+    ]
+    services_mapping.text_fields_set = set(['someField', 'otherField'])
```

Comment:
My IDE always complains that this should be a set literal, `{'someField', 'otherField'}`, so would be nice to change it to that. :)